src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -265,12 +265,6 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
-                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
Why is not suppressed? If it is not configurable then it should not be configurable by any means - so something that's off by default and not configurable should never be able to be turned on. That's not entirely useful but that's consistent and keeps the meaning of the tag simple.